bin/blin.p6

```diff
@@ -169,14 +171,22 @@ for @sources {
     my $json-data = run(:out, <curl -->, $_).out.slurp;
     my $json = from-json $json-data;
     for @$json {
+        my $dist = try Zef::Distribution.new(|$_);
+        my $depends = $dist ??
+            ([∪]
+             ($dist.depends-specs.map(*.identity)      ).Set,
+             ($dist.test-depends-specs.map(*.identity) ).Set,
+             ($dist.build-depends-specs.map(*.identity)).Set) ∖ $ignored-deps
```

Comment:
This is all mostly correct. However, now you really want to know if $ignored-deps matches a given spec (not just string match on the full name) since .identity will give like `Foo:ver<*>:api<*>:auth<>` which isn't going to string match against `Foo` or `Foo:ver<1>`. https://gist.github.com/ugexe/89c9df9209d56d38194c74d457330679 was a draft of what I thought might work with the key bits being:

```
+            state @ignore-specs = $ignored-deps.keys.map({ Zef::Distribution::DependencySpecification.new($_) });
+            my @depends =
+                map  -> $spec { $spec.identity }
+                grep -> $spec { not @ignore-specs.grep({ $_.matches-spec($spec) }) }
+                slip($dist.depends-specs),
+                slip($dist.test-depends-specs),
+                slip($dist.build-depends-specs),
+            ;
```